tdvt/TDVT Results.twb

```diff
@@ -3895,1687 +3559,198 @@ Error while execut&quot;</bucket>
       </viewpoint>
       <simple-id uuid='{FF3BB016-DBC3-4046-80CE-99C64F4FFD07}' />
     </window>
-    <window class='worksheet' hidden='true' name='Test Details' tab-color='#f66273'>
-      <cards>
-        <edge name='left'>
-          <strip size='152'>
-            <card type='pages' />
-            <card type='filters' />
-            <card type='marks' />
-          </strip>
-        </edge>
-        <edge name='top'>
-          <strip size='2147483647'>
-            <card type='columns' />
-          </strip>
-          <strip size='2147483647'>
-            <card type='rows' />
-          </strip>
-          <strip size='8'>
-            <card type='title' />
-          </strip>
-        </edge>
-        <edge name='right'>
-          <strip size='160'>
-            <card pane-specification-id='0' param='[textscan.1gc4mna00d9z6u19vsqbk09t94y2].[none:Priority:nk]' type='color' />
-          </strip>
-        </edge>
-      </cards>
-      <viewpoint>
-        <zoom type='fit-width' />
-        <highlight>
-          <color-one-way>
-            <field>[textscan.1gc4mna00d9z6u19vsqbk09t94y2].[none:Actual (100)tuples:nk]</field>
-            <field>[textscan.1gc4mna00d9z6u19vsqbk09t94y2].[none:Actual tuples:nk]</field>
-            <field>[textscan.1gc4mna00d9z6u19vsqbk09t94y2].[none:Calculation_1487173092637745153:nk]</field>
-            <field>[textscan.1gc4mna00d9z6u19vsqbk09t94y2].[none:Calculation_1487173092642668549:nk]</field>
-            <field>[textscan.1gc4mna00d9z6u19vsqbk09t94y2].[none:Calculation_1487173092652093447:nk]</field>
-            <field>[textscan.1gc4mna00d9z6u19vsqbk09t94y2].[none:Calculation_3156952999011225602:nk]</field>
-            <field>[textscan.1gc4mna00d9z6u19vsqbk09t94y2].[none:Calculation_921830567081418753:nk]</field>
-            <field>[textscan.1gc4mna00d9z6u19vsqbk09t94y2].[none:Error Msg:nk]</field>
-            <field>[textscan.1gc4mna00d9z6u19vsqbk09t94y2].[none:Expected (100)tuples:nk]</field>
-            <field>[textscan.1gc4mna00d9z6u19vsqbk09t94y2].[none:Expected tuples:nk]</field>
-            <field>[textscan.1gc4mna00d9z6u19vsqbk09t94y2].[none:Passed (copy):ok]</field>
-            <field>[textscan.1gc4mna00d9z6u19vsqbk09t94y2].[none:Passed:nk]</field>
-            <field>[textscan.1gc4mna00d9z6u19vsqbk09t94y2].[none:Priority:nk]</field>
-            <field>[textscan.1gc4mna00d9z6u19vsqbk09t94y2].[none:Suite:nk]</field>
-            <field>[textscan.1gc4mna00d9z6u19vsqbk09t94y2].[none:TDSName:nk]</field>
-            <field>[textscan.1gc4mna00d9z6u19vsqbk09t94y2].[none:Test Case:nk]</field>
-            <field>[textscan.1gc4mna00d9z6u19vsqbk09t94y2].[none:Test Type:nk]</field>
-            <field>[textscan.1gc4mna00d9z6u19vsqbk09t94y2].[none:TestName:nk]</field>
-          </color-one-way>
-        </highlight>
-      </viewpoint>
-      <simple-id uuid='{CA2BEB56-0251-4E7A-926F-9F49C0B3B673}' />
-    </window>
   </windows>
   <thumbnails>

```

Comment:
I think we can delete the `thumbnails` element or at least its children for this type of shared twb use case.